DaaS for staging systems [RN_38087] (#2356)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.

#### 4 December 2023 - New Feature - DataMiner as a Service for staging systems [ID_XXXXXXXXXXXXXXXXXXX]

From now onwards it is possible to create a DataMiner as a Service (DaaS) system on dataminer.services for staging systems. Read more about [DataMiner as a Service](https://docs.dataminer.services/user-guide/Getting_started/Creating_a_DMS_in_the_cloud.html) or about [DataMiner Express](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#dataminer-express).

> Deploying such a DaaS system will deduct 3 DataMiner credits from the organization every week onwards as [Pay-per-Use](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#pay-per-use). In case the organization runs out of DataMiner credits, the particular DaaS system will be deleted. By default every organization is provided with 3 DataMiner credits, so a DaaS system can be tried out for one week free of charge.

Reviewer: ```suggestion
Deploying such a DaaS system will deduct 3 DataMiner credits from the organization every week onwards as [Pay-per-Use](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#pay-per-use). In case the organization runs out of DataMiner credits, the expired DaaS system will be deleted. By default every organization is provided with 3 DataMiner credits, so a DaaS system can be tried out for one week free of charge.
```

> When created a DaaS system, your dataminer.services account will automatically be linked to your DataMiner account, so you can easily access the web apps like Monitoring via remote access.

Author: ```suggestion
When creating a DaaS system, your dataminer.services account will automatically be linked to your DataMiner account, so you can easily access the web apps like Monitoring via remote access.
```

The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.

#### 4 December 2023 - New Feature - DataMiner as a Service [ID_XXXXXXXXXXXXXXXXXXX]

> From now onwards it is possible to create a DataMiner System as a Service on dataminer.services, in short called DaaS. 

Reviewer: ```suggestion
From now onwards it is possible to create a DataMiner System as a Service (DaaS) system on dataminer.services.
```

> Deploying such a DaaS will deduct 3 DataMiner credits from the organization. Every week onwards another 3 DataMiner Credits will be deducted unless the organization runs out of DataMiner Credits. In that case the particulaar DaaS instance will be deleted.

Reviewer: ```suggestion
Deploying such a system will deduct 3 DataMiner credits from the organization. Every week onwards another 3 DataMiner Credits will be deducted unless the organization runs out of DataMiner Credits. In that case the particular DaaS system will be deleted.
```

> When a DaaS is created, the dataminer.services account and the DataMiner account will automatically be linked to each each other, so the web apps like Monitoring are easily available.

Reviewer: Not sure if it's worth mentioning here, it'd be if this previously wasn't the case. But this is the for change log for DaaS

> A DaaS can be manually deleted easily from dataminer.services like you would for any other DMS.

Reviewer: ```suggestion
A DaaS system can be manually deleted easily from dataminer.services like you would for any other DMS.
```
